lib/default_cpolicy.json

```diff
@@ -0,0 +1,92 @@
+{
+	"rules": 
```

Comment:
anyway, this does not belong in this folder, it belongs in the opencog/reasoning/pln folder